proto/provenance/expiration/v1/events.proto

```diff
@@ -0,0 +1,55 @@
+// Since: provenance 1.12
```

Comment:
Not accurate. Also, I'm not a fan of the "since ..." comments that the sdk has started using everywhere; I just don't see them as useful.